src/main/java/mate/zorii/bookstore/service/impl/UserServiceImpl.java

```diff
@@ -23,7 +26,12 @@ public UserResponseDto register(UserRegistrationRequestDto bookDto)
             throw new RegistrationException("Email is already registered");
         }
         User user = userMapper.toModel(bookDto);
-        userRepository.save(user);
+        User savedUser = userRepository.save(user);
+
+        ShoppingCart shoppingCart = new ShoppingCart();
+        shoppingCart.setUser(savedUser);
+        shoppingCartRepository.save(shoppingCart);
+
```

Comment:
```suggestion
        userRepository.save(user);
        shoppingCartService.createShoppingCart(user);
```

Where in ShoppingCartServiceImpl

```
@Override
    public void createShoppingCart(User user) {
        ShoppingCart shoppingCart = new ShoppingCart();
        shoppingCart.setUser(user);
        repository.save(shoppingCart);
    }
```